feature/Github-Label/app.js

```diff
@@ -0,0 +1,43 @@
+const express = require("express");
+const mongoose = require("mongoose");
+
+// Require routes
```

Comment:
These comments don't add any value as code is self-explanatory. 
Unless something is not explicitly clear we don't add comments.

Reply:
Now its resolved @harshaanNihal